x-pack/plugins/cases/server/common/error.ts

```diff
@@ -28,7 +28,7 @@ class CaseError extends Error {
    * and data from that.
    */
   public boomify(): Boom<unknown> {
-    const message = this.message ?? this.wrappedError?.message;
+    const message = this.wrappedError?.message ?? this.message;
```

Comment:
I think this makes sense 🤔  Basically when we `boomify()` we want the original boom message instead of the concatenated one from the various call stacks. This way we keep the message in the API response lean and clean and don't have the repeated messages concatenated together. The `boomify()` is currently only used the HTTP routes.

I think the reason this wasn't failing before is because by moving the logic to the client I added an additional `try/catch` which wasn't there before so all the integration tests were passing before.

Reply:
Isn't `this.message` the original one? Inside `createCaseError` the case error is being created as `new CaseError(message, error);`. I think I am missing something 😄

Reply:
Haha, yeah it is kind of confusing. `this.message` is the message passed to the `createCaseError` function which will be the _new_ message. `this.wrappedError.message` is the original error (aka the message from the `Boom.*('whatever')`